app/models/school_mentor.rb

```diff
@@ -5,5 +5,6 @@ class SchoolMentor < ApplicationRecord
   belongs_to :participant_profile, class_name: "ParticipantProfile::Mentor"
   belongs_to :preferred_identity, class_name: "ParticipantIdentity"
 
+  scope :by, ->(participant_profile) { where(participant_profile:) }
```

Comment:
Not sure of the value of this as it is only a simple `where(participant_profile:)` and is a bit ambiguous, what if I want want to look up `by` school?  I get it's adding a bit of sugar but it then also becomes something else to remember.

Reply:
I agree it is a simple scope just filtering by participant. My idea was not to have to take the `where(participant_profile:)` out of the SchoolMentor model. I think this is something internal to the class.

Should renaming it to something more meaningful do the job?. When I wrote it I was not 100% happy with any name:
`by(participant_profile)`
`by_participant_profile(participant_profile)`
`by_mentor(participant_profile)`
`by_participant(participant_profile)`
 
Any suggestion?

Reply:
Ok, code rewritten without the scope.